File: app/src/main/java/com/edorin/kotlindemo/java/MainActivity.java
package com.edorin.kotlindemo.java;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Spinner;
import android.widget.TextView;
import com.edorin.kotlindemo.R;

public class MainActivity extends AppCompatActivity {

    String title = "University Of Assumption";

    TextView txvTitle;
    EditText edtFirstname, edtLastname;
    Spinner spnGender;
    Button btnSubmit;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // initialize widgets
        txvTitle = findViewById(R.id.txvTitle);
        edtFirstname = findViewById(R.id.edtFirstname);
        edtLastname = findViewById(R.id.edtLastname);
        spnGender = findViewById(R.id.spnGender);
        btnSubmit = findViewById(R.id.btnSubmit);

        // set title text
        txvTitle.setText(title);

        // click listener
        btnSubmit.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent();
                intent.putExtra("firstname", edtFirstname.getText());
                intent.putExtra("lastname", edtLastname.getText());
                intent.putExtra("gender", spnGender.getSelectedItem().toString());
                startActivity(intent);
            }
        });

    }

}

File: app/src/main/java/com/edorin/kotlindemo/kotlin/MainActivity.kt
package com.edorin.kotlindemo.kotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.edorin.kotlindemo.R
import com.edorin.kotlindemo.java.SummaryActivity
import com.edorin.kotlindemo.selectedItemString
import kotlinx.android.synthetic.main.activity_main.*
import org.jetbrains.anko.startActivity

class MainActivity : AppCompatActivity() {

    val title = "University of Assumption"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // set title text
        txvTitle.text = title

        // click listener
        btnSubmit.setOnClickListener {
            startActivity<SummaryActivity>(
                    "firstname" to edtFirstname.text,
                    "lastname" to edtLastname.text,
                    "gender" to spnGender.selectedItemString())
        }
    }

}



